src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -1530,7 +1530,7 @@ internal override ImmutableArray<Symbol> GetEarlyAttributeDecodingMembers(string
 
                 if (!membersAndInitializers.HaveIndexers)
                 {
-                    membersByName = membersAndInitializers.NonTypeMembers.ToDictionary(s => s.Name);
+                    membersByName = membersAndInitializers.NonTypeMembers.ToDictionary(static s => s.Name);
```

Comment:
these other callers weren't passing a Comparer.  So the one place we pass seems suspect.  Removing that to see if anything actually breaks.